### PR TITLE
New package: apfel

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/apfel/package.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from spack import *
+
+
+class Apfel(AutotoolsPackage):
+    """APFEL is a library able to perform DGLAP evolution up to NNLO in QCD and
+       to NLO in QED, both with pole and MSbar masses. The coupled DGLAP
+       QCD+QED evolution equations are solved in x-space by means of higher
+       order interpolations and Runge-Kutta techniques."""
+
+    homepage = "http://apfel.hepforge.org/"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/apfel/package.py]: Why are you not using the URL on the GitHub page as the homepage (i.e., https://apfel.mi.infn.it/)?

(My browser does not like the URL you provided:  "..connection is not private..".)

Comment (iarspider) [reply var/spack/repos/builtin/packages/apfel/package.py]: The .it URL is not a homepage, rather a live demo. I can change the URL to their github (https://github.com/scarrazza/apfel) if you prefer it that way. I have also sent an e-mail to the developers asking them to update the SSL certificate for the hepforge site.

**Review by tldahlgren — APPROVED** [review]
LGTM.

**Comment by tldahlgren** [comment]
Thank you for your contribution!

**Pull request merged**